### ISSUE TITLE
Total Terminal Compatibility

### ISSUE THREAD

**Issue by nickw** [achiurizo/consular-osx#2]
Just came across an issue trying to use this with [TotalTerminal](http://totalterminal.binaryage.com/). It's complaining about active_window being nil. I changed:

`window.properties_.get[:frontmost] rescue false`

to

`window.properties_.get[:frontmost] rescue windows.first`

and it's happy now. What do you think of this change - should I create a pull request for it?

**Comment by achiurizo** [comment]
@nickw yeah send a pull request. Did you see if it breaks anything else? I'll check it again too when you sent the request.
